packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,240 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import Anatomy from './ActionBarAnatomy.svg';
+import docs from 'docs:@react-spectrum/actionbar';
+import {HeaderInfo, PropTable, PageDescription, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/actionbar/package.json';
+
+```jsx import
+import {ActionBar, ActionBarContainer, Item} from '@react-spectrum/actionbar';
+```
+
+---
+category: Collections
```

Comment:
Is Collections the right category (felt most closely related to those components)? Maybe Overlays?

Reply:
Sounds good, I'll move it to Overlay

Reply:
hmmm but this isn't an overlay, at least, not in the classic sense. It's rendered inline and space is made for it using flex.
I think I might consider a different category, a new one, something like a companion, container, wrapper, extra functionality, bulk actions, collection actions, actions, .... naming is hard, haha

Reply:
Thats a good point, I had figured that it looked like an Overlay although not truly being one and thus people would look for it in that category there. I like "Collection Actions" personally, open to other options

Reply:
I feel like this is similar to an ActionMenu on a card, which would be a collection item action. I'm trying to figure out what else other things like this would be.